### PR TITLE
Apposite 20487 d mod08 lak

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Comments inline.

### CODE REVIEW

File: Instructions/20487D_MOD08_LAK.md [hunk 1]
```diff
@@ -1,234 +1,234 @@
...
 #### Task 2: Record .NET ETW events in PerfView
 
-1. Open **PerfView**
-2. Click on **Collect** in top menu bar.
-3. Click on **Start Collection**.
+1. Open PerfView.
+2. In the top menu bar, click **Collect**.
+3. Click **Start Collection**.
 
 #### Task 3: Run a script to invoke service and meck it to throw exceptions
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Should this be "make" instead of "meck"?

File: Instructions/20487D_MOD08_LAK.md [hunk 2]
```diff
@@ -1,234 +1,234 @@
...
-5. Double click on **Throw(System.Exception) Id {number} can't be hash to flight code** in the column **Name**.
+1. Double-click **PerfViewData.etl.zip**.
+2. Expand the **Advanced Group** folder, and then double-click **Exceptions Stacks**.
+3. In the **Select Process** window, locate the following process:
+   - In the **Name** column, find **dotnet**.
+   - In the **CommandLine** column, find **dotnet exec** *[Repository Root]***\AllFiles\Mod08\Labfiles\Lab1\Blueyonder.Service**.
+   - Double-click the process.
+4. In the **Exception Stacks** window, click the **By Name** tab.
+5. In the **Name** column, double-click **Throw(System.Exception) Id** *{number}* **can't be hash to flight code**.
 6. View the exception call stack see **Blueyonder.Service.Controllers.FlightsControllers.Get(int32)** the service that throw the exception.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Instruction not clear.

File: Instructions/20487D_MOD08_LAK.md [hunk 3]
```diff
@@ -1,234 +1,234 @@
...
-7. In **GroupPats** selectlist select **[no grouping]** on the top bar.
-8. Expand **Blueyonder.Service.Repository.FlightsRepository.HashFlightCode(int32)**
-9. View exception call stack to see which method in the service throw the exception.
-   > **Note:** **HashFlightCode** method throw exception and continue to **GetFlightCode**.
+7. On the top bar, in the **GroupPats** list, select **[no grouping]**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Please verify my edit.

File: Instructions/20487D_MOD08_LAK.md [hunk 4]
```diff
@@ -1,234 +1,234 @@
...
 
 1. Open **Azure Portal**.
-2. Click on **All resources** then click on **blueyondermod08lab2**{YourInitials}.
-3. Click on **Application Insights** in the **SETTINGS** section on the left menu.
-4. Click on **Setup Application Insights** then add the following information:
-    >**Note:** First time need to click on **Setup Application Insights** link.
+2. Click **All resources**, and then click **blueyondermod08lab2***{YourInitials}*.
+3. In the left menu, in the **SETTINGS** section, click **Application Insights**.
+4. Click **Setup Application Insights**, and then add the following information:
+    >**Note**: First time need to click on **Setup Application Insights** link.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Instruction not clear. This note seems to say the same thing as the step.

File: Instructions/20487D_MOD08_LAK.md [hunk 5]
```diff
@@ -1,234 +1,234 @@
...
+5. In the **Application Insights** blade, click **View more in Application Insights**.
 6. Copy **Instrumentation Key**.
-7. Open **Command Line**.
-8. Run the following command to change directory to **Starter** folder:
+7. Open the command prompt.
+8. To change directory to the **Starter** folder, run the following command:
     ```bash
     cd [Repository Root]\Allfiles\Mod08\Labfiles\Lab2\Starter
     ```
 9.  Run the following command to install :
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: To install what?

File: Instructions/20487D_MOD08_LAK.md [hunk 6]
```diff
@@ -249,53 +249,59 @@
...
-    - In **Subscription** click on  **Azure Pass**.
-    - Click on **OK**.
-5. Click on **All resources** then click on **blueyondervsts**.
-6. Click on **Url** value to naviage to **VSTS**.
-7. Login with credential in **VSTS**
-8. In **Create new project**  add the following information to create new project in **VSTS**:
-    - In **Project name** type **Blueyonder**.
-    - Click on **Create Project**.
+1. Switch to Azure Portal.
+2. Click **All resources**, and then click **blueyondermod08lab2***{YourInitials}* type of **Application Insights**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Instruction not clear.

File: Instructions/20487D_MOD08_LAK.md [hunk 7]
```diff
@@ -249,53 +249,59 @@
...
-    - Click on **CONFIGURE TEST USING**:
-        - In **TEST TYPE** select **Manual Test**.
-        - In **URL** type **http://blueyondermod08lab2{YourInitials}.azurewebsites.net/api/destinations**.
-        - Click on **Done**.
-    - In **Name** type **DestinationsTest**.
-    - In **USER LOAD** type **20**.
-    - In **DURATION (MINUTES)** type **5**.
-    - Click on **Run test**.
+1. Switch to Azure Portal.
+2. Click **All resources**, and then click **blueyondermod08lab2***{YourInitials}* type of **Application Insights**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Instruction not clear.

File: Instructions/20487D_MOD08_LAK.md [hunk 8]
```diff
@@ -249,53 +249,59 @@
...
-1. Switch to **VSTS**.
-2. Hover over **Test Plans** then click on **Load test** on the top menu bar.
-3. Double click on **DestinationsTest**.
-4. View the Summary of the test.
-5. Click on **Charts** tab to view the **Performance** of the test.
+1. Switch to VSTS.
+2. On the top menu bar, place the mouse pointer over **Test Plans**, and then click **Load test**.
+3. Double-click **DestinationsTest**.
+4. View the summary of the test.
+5. To view the **Performance** of the test, click the **Charts** tab.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD08_LAK.md]: Should this be "Performance Report"?

File: Instructions/20487D_MOD08_LAK.md (hunk 3 above)
Comment (AviAvni) [reply Instructions/20487D_MOD08_LAK.md]: ok

**Pull request merged**